Classes/Hook/UserSettings.php

```diff
@@ -205,6 +205,9 @@ private function getSecretKey(): string
 
     private function isGoogleAuthenticatorEnabled(): bool
     {
+        if ($this->data['type'] === 'user' && !is_array($this->data['row'])) {
+            $this->data['row'] = $GLOBALS['BE_USER']->user;
```

Comment:
Codacy found an issue: [Direct use of $GLOBALS Superglobal detected.](https://app.codacy.com/manual/robinvonberg/cf_google_authenticator/pullRequest?prid=5757557)

Reply:
IMO there's not really any better way. https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/BackendUserObject/Index.html